octoprint_mrbeam/camera/corners.py

```diff
@@ -25,7 +25,7 @@
 _logger = mrb_logger("octoprint.plugins.mrbeam.camera.corners")
 
 # @logtime()
-def warp_image_by_corners(image, corners, zoomed_out=False):
+def warpImgByCorners(image, corners, zoomed_out=False):
```

Comment:
Why did we change the name here? It doesn't follow the convention...

Reply:
We changed this in a refactor a while back and I just found out that this is used by the camera plugin and the plugin is failing to load because of it so I reverted the name

Reply:
Aaaah okay! Could you please add a comment so we don't do it again?

Reply:
done